src/Data/String/LineWrapping.purs

```diff
@@ -106,5 +114,5 @@ printWrappedLine =
     SingleWordWrappedLine x →
       printMeasuredWord x
     MultipleWordWrappedLine x1 x2 xs →
-      String.joinWith " " $ printMeasuredWord <$> [x1] <> [x2] <> xs
+      String.joinWith "\n" $ printMeasuredWord <$> [x1] <> [x2] <> xs
```

Comment:
Still could do the `[x1, x2] <> xs` thing here - also, is that change to `"\n"` in the `joinWith` right? I don't really understand this constructor to be honest.